### DISCUSSION TITLE
408 During Transcription

### DISCUSSION THREAD

**Discussion by chasemcdo** [deepgram/community#1007] [General help]
Been having the occasional issue with a 408 time out response when trying to transcribe audio. I just had this happen on around 8 back to back transcriptions.

Is there anyway to get around or mitigate these errors?

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Reply by chasemcdo** [reply]
done

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- A request ID that triggered your error or issue.

**Reply by chasemcdo** [reply]
- Python is where we are initiating from
- and here are a few ids:
  - 5f0b1abc-6647-4ffa-85ce-6d858345f363
  - 944a0908-fd5f-422c-9350-3c2b4fd98913
  - e2a23d06-bfff-4859-91d7-45aeb32190d0
